operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -3,24 +3,41 @@
 The Configuration Framework Service \(CFS\) enables Ansible playbooks to run against both running nodes \(node personalization\) and images prior to boot\(image customization\).
 See [Configuration Management Use Cases](Configuration_Management.md#use-cases) for more information about image customization and when it should be used.
 
-Ideally image customization playbooks should be separate from node personalization playbooks. This reduces the likelihood that Ansible content will run in both modes, and reduces the need for conditional checks in the playbooks.
+## Using the `cfs_image` host group
 
-For situations when both image customization and node personalization need to be part of the same playbook, CFS provides the `cray_cfs_image` variable to distinguish between
-node personalization and image customization. When this variable is set to `true`, it indicates that the CFS session is an image customization and the playbook is targeting an image.
+During image customization, CFS will automatically add all image customization hosts into a special `cfs_image` host group in Ansible inventory.
+Plays intended for image customization can then target this group in addition to any other provided host groups.
 
-## Using `cray_cfs_image`
+To target only image customization, plays should use the following syntax.  In this example the play is targeting only _images_ for `Compute` nodes. `&` takes the intersection of the `Compute` and `cfs_image` groups.
```

Comment:
Perhaps linking to this part of the Ansible docs is helpful here? https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html#common-patterns